examples/adaptive-cards/README.md

```diff
@@ -0,0 +1,16 @@
+<!--- Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. See the NOTICE file distributed with this work for additional information regarding copyright ownership. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+--->
+
+# ComposeUI Default App
+
+This is a simple webpage that loads adaptive-cards and showcases a button which launches an alert when clicked.
+
+## During Development
+
+```npm i
+
+npx lerna run start --stream --scope=@morgan-stanley/composeui-example-adaptive-cards
+```
+
+<!---
+composeui .\Tryouts\Plugins\ApplicationPlugins\defaultApp\app.js --->
```

Comment:
What is this comment here?

Reply:
It was part of the default app. It's been removed.